views.go

```diff
@@ -496,7 +347,7 @@ func (lk *logKeeper) viewBuild(w http.ResponseWriter, r *http.Request) {
 	}{build, tests}, "base", "build.html")
 }
 
-func (lk *logKeeper) viewBucketBuild(r *http.Request, buildID string, shouldFallBack bool) (*model.Build, []model.Test, *apiError) {
+func (lk *logKeeper) viewBucketBuild(r *http.Request, buildID string) (*model.Build, []model.Test, *apiError) {
```

Comment:
could it make sense for this to also return a `fallback` boolean?

Reply:
This doesn't really need it since we always want to fallback with a 404. When fetching logs only want to fall back if we get a 404 fetching the build, if we get a 404 for the test then we don't want to fall back since that we means we found the build in S3 but the test does not exist.